.evergreen.yml

```diff
@@ -138,59 +183,52 @@ tasks:
 
   - name: e2e_test_feature_compatibility_version
     commands:
-      - func: clone
-      - func: setup_kubernetes_environment
       - func: run_e2e_test
         vars:
           test: feature_compatibility_version
+  #    - func: upload_e2e_logs
```

Comment:
no uploading of logs for now?

Reply:
This was a comment left by mistake, the upload has been moved into a teardown task